src/calculation/HTMLBuilder.ts

```diff
@@ -92,23 +92,40 @@ Handlebars.registerHelper('highlightCoverage', (localId, context) => {
  */
 export function sortStatements(measure: fhir4.Measure, groupId: string, statements: StatementResult[]) {
   const group = measure.group?.find(g => g.id === groupId) || measure.group?.[0];
-  const populationSet = new Set(group?.population?.map(p => p.criteria.expression));
+  const populationOrder = Object.values(PopulationType);
```

Comment:
hmmmm. Not sure I like relying on the iteration order of `Object.values` for a TypeScript enum. I think we should bring back the explicit ordering in a constant array.